### DISCUSSION TITLE
Newbie

### DISCUSSION THREAD

**Discussion by Rohith-repo** [community/community#131537] [New to GitHub]
Hi Guys I'm Rohith and Im new to this community 👍🗣️

**Comment by ghostinhershell** [comment]
👋 Welcome to the GitHub Community on Discussions, @Rohith-repo! Happy to have you here 🙂

To get started, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776).

**Comment by Dimash357** [comment]
response 1
